apps/dashboard/app/views/widgets/pinned_apps/_app.html.erb

```diff
@@ -1,26 +1,32 @@
 <%- link = app.links.first -%>
 <%- link_data = {:method => "get"}.merge link.data -%>
+<%- tile_data = link.tile -%>
+<%- border_color = tile_data[:border_color] ? "border-color: #{tile_data[:border_color]};" : "" -%>
+
 <div class="col-sm-3 col-md-3 app-launcher-container">
   <div class="app-launcher app-launcher-hover" data-toggle="launcher-button">
   <%=
     link_to(
       link.url.to_s,
       class: "",
       target: link.new_tab? ? "_blank" : nil,
-      data: link_data
+      data: link_data,
+      style: border_color
```

Comment:
Would css classes like `red` and `blue` work?  We're trying to remove inlining `style`s for content security policy (CSP) reasons. I.e., there's a rule not to use inline styles.

Reply:
Yea I confirmed this would be blocked by `style-src 'self' 'nonce-...;`. Since we use SASS we can programmatically generate some classes at compile time for more colors.

Reply:
Ah, makes sense.
I have updated to use CSS classes for border color override.

We could add around 10 CSS classes for the 10 most common/popular colors. This would help most of the admins that want custom colors, but do not want to add custom CSS.

Reply:
Added some CSS classes to override the border color based on the core colors from bootstrap.
This is just as a reference